Shared/Samples/Change viewpoint/ChangeViewpointView.swift

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 Esri
```

Comment:
```suggestion
// Copyright 2023 Esri
```
Does this need to be changed?